backend/src/project/service.ts

```diff
@@ -58,8 +58,22 @@ export const getAllProjectsWithAuthor = async (user?: User): Promise<ProjectLike
     const id = new ObjectId(user?._id);
 
     const result = await projects
-        .aggregate([canUserReadProjectDBQuery(id), ...authorLookup])
-        .sort({ modificationDate: -1 })
+        .aggregate([
+            canUserReadProjectDBQuery(id),
+            ...authorLookup,
+            {
```

Comment:
Przeniósłbym to do osobnej zmiennej tak jak `canUserReadProjectDBQuery` i `authorLookup`

Reply:
Chyba się nie zrozumieliśmy w tej kwestii i podejrzewam, że to dlatego, że github trochę myląco wyświetla te komentarze, bo wyświetla kawałek kodu nad komentarzem, a ja odnoszę się do kodu znajdującego się pod komentarzem. Więc tutaj nie chodziło mi o zmianę `canUserReadProjectDBQuery(id)` i `authorLookup`, tylko o ten fragment:
`{$addFields: {
                    isAuthor: {
                        $cond: {
                            if: { $eq: ["$authorId", userId] },
                            then: 1,
                            else: 0,
                        },
                    },
                },
            }`
żeby to wyeksportować do osobnej zmiennej. 

Tamte zmienne, które dodałeś w poprzednim comicie (`query` i `lookup`) usuń i przywróc to co było wcześniej bo to nie o to chodziło.